pallets/identity/src/lib.rs

```diff
@@ -1995,25 +1989,23 @@ impl<T: Trait> Module<T> {
     ) -> bool {
         match authorization_data {
```

Comment:
```suggestion
        type_of_auth == match authorization_data {
```
(factor out equality check from each branch)